server/app/repository/ProgramRepository.java

```diff
@@ -346,8 +346,6 @@ public PaginationResult<ApplicationModel> getApplicationsForAllProgramVersions(
             .setLabel("ApplicationModel.findList")
             .setProfileLocation(
                 queryProfileLocationBuilder.create("getApplicationsForAllProgramVersions"))
-            .fetch("program")
-            .fetch("applicant")
```

Comment:
re Defensive Programming™️ :
Is it worth still fetching these, but with `fetchLazy()`?

This would mean that if someone updated the code to access properties on the program or applicant, it would at least batch the additional queries instead of a full n+1 situation, but, iiuc, it wouldn't ever actually do the fetch with the code the way it is now cause we don't access any properties except the ids

Reply:
That seems like a good approach if we expect to access those fields!

Reply:
We don't currently, but I could see someone not realizing and accidentally accessing them in the future.